source/includes/_tx.md

```diff
@@ -2186,6 +2186,20 @@ payload = JSON.parse('{ "inputs": [{
 	}] }')
 ```
 
+```python
+from blockcypher import create_unsigned_tx, make_tx_signatures, broadcast_signed_transaction
+# Creating unsigned transaction:
+pubkey_list = [pubkey1, pubkey2, pubkey3]
+inputs = [{'pubkeys':pubkey_list, 'script_type': 'multisig-2-of-3'}]
+outputs = [{'address': dest_address, 'value':10000000 }]
+
+unsigned_tx = create_unsigned_tx(inputs=inputs, outputs=outputs, coin_symbol='bcy')
+# signing with 2 of the 3 private keys (it's possible to sign only with one of them in a first phase, and broadcasting it. The transaction will be validated only when the network will receive 2 signatures):
+tx_signatures = make_tx_signatures(txs_to_sign=unsigned_tx['tosign'], privkey_list=[privkey1,privkey3], pubkey_list=[pubkey1,pubkey3])
```

Comment:
Thanks @gustavklopp. Are you sure this will work? It's hard to say for sure since there's not real data here, but I'm pretty sure this will fail.

`make_tx_signatures` loops through each entry in `txs_to_sign` and signs it using the nth item in `privkey_list`. To use `make_tx_signatures` with multisig requires passing in each `tx_to_sign` in duplicate as many times as it's going to be signed. My example from our thread is here:
https://github.com/blockcypher/blockcypher-python/issues/28#issuecomment-171719835

```
make_tx_signatures(
    txs_to_sign['35fc0f055ec9145631c6216c6cbba3d2e6f75b061b4bfb828a530c4788393608', '35fc0f055ec9145631c6216c6cbba3d2e6f75b061b4bfb828a530c4788393608']
    privkey_list=[privkey1, privkey2],
    pubkey_list=[pubkey1, pubkey2],
)
```

Note that it's not just as easy as duplicating txs_to_sign, it's possible that there could be multiple UTXOs each encumbered by multiple p2sh addresses! Perhaps I should supply a new `make_tx_signatures` function that takes a list of private keys and handles the logic of figuring out which inputs to sign with which private keys, and then does the signing. As-is it's much more complicated. Not trivial though.